core/test/accessor/CMakeLists.txt

```diff
@@ -14,7 +14,9 @@ function(create_accessor_test test_name)
     set_target_properties("${TEST_TARGET_NAME}" PROPERTIES
         OUTPUT_NAME "${test_name}")
     target_link_libraries("${TEST_TARGET_NAME}" PRIVATE GTest::Main GTest::GTest ${ARGN})
-    add_test(NAME "${REL_BINARY_DIR}/${test_name}" COMMAND "${TEST_TARGET_NAME}")
+    add_test(NAME "${REL_BINARY_DIR}/${test_name}"
+        COMMAND "${TEST_TARGET_NAME}"
+        WORKING_DIRECTORY "$<TARGET_FILE_DIR:ginkgo>")
```

Comment:
Why set the working directory here, that would make it be `build/core` if I understand this directive?

Reply:
There is a longer story behind this which I should mention: Contrary to the CMake default, GTest builds static libraries by default, if BUILD_SHARED_LIBS is unspecified. That means that previously, all tests were linked statically in all settings and we didn't need to worry about the location. With ExternalProject, we didn't forward our BUILD_SHARED_LIBS settings, but with add_subdirectory we do, which means we are now linking dynamically to GTest.
To make tests work on Windows without having to copy around DLLs all the time, we placed all libraries and DLLs into ´lib/`, which is also where `libginkgo.dll` and more importantly `libgtest.dll` will be stored. So this change is necessary for all tests to find `libgtest.dll`

Reply:
Thanks for the detailed explanation. If that's the case, maybe it makes sense to explicitly add an `if (WIN32)` to highlight that fact, and keep the defaults otherwise?

Reply:
The working directory does not matter except for dynamic linking, so I would prefer to keep this, as the explicit conditional would make the code more complex.